<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>411</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>user</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>query</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>331</width>
        <height>251</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>单词1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>单词2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_7">
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>51</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>词库1</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>51</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>40</width>
          <height>12</height>
         </rect>
        </property>
        <property name="text">
         <string>learn</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_8">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>191</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>词库初始界面，需要利用词库数据初始化</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_3">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>250</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_9">
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>151</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>进一步的，练习/学习界面</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_4">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>278</y>
          <width>41</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>review</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>40</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>ach</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>game</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QToolButton" name="btn_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>用户</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_query">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_learn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>学习</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_review">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>复习</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_achievement">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>成就</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_game">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>游戏</string>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
